notebook/static/notebook/js/codecell.js

```diff
@@ -385,7 +385,12 @@ define([
      * @private
      */
     CodeCell.prototype._handle_set_next_input = function (payload) {
-        var data = {'cell': this, 'text': payload.text, replace: payload.replace};
+        var data = {
+            cell: this,
```

Comment:
JS only requires quoting keys when they are not valid identifiers.

On Tue, Dec 15, 2015 at 11:30 AM, Carol Willing <notifications@github.com>
wrote:

> In notebook/static/notebook/js/codecell.js
> <https://github.com/jupyter/notebook/pull/875#discussion_r47685184>:
>
> > @@ -385,7 +385,12 @@ define([
> >       * @private
> >       */
> >      CodeCell.prototype._handle_set_next_input = function (payload) {
> > -        var data = {'cell': this, 'text': payload.text, replace: payload.replace};
> > +        var data = {
> > +            cell: this,
>
> @minrk <https://github.com/minrk> Just curious to improve my own JS...why
> did you drop the single quotes around *cell* and *text* here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/875/files#r47685184>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com